src/lib/rest/RestService.cpp

```diff
@@ -138,7 +140,14 @@ static std::string tenantCheck(const std::string& tenant)
 
   if (strlen(name) > SERVICE_NAME_MAX_LEN)
   {
-    LM_W(("Bad Input (a tenant name can be max %d characters long. Length: %d)", SERVICE_NAME_MAX_LEN, strlen(name)));
+    char numV1[16];
```

Comment:
Not really a 'limit'.
That was not my intention with the file - I thought more about limitations of the broker, line default-pagination-size and stuff like that.

This is simply "big enough to hold any integer".
Using a define only makes the while thing more difficult to understand.
As always, we have different opinions on this ...